Gemfile

```diff
@@ -90,7 +90,7 @@ gem 'cancancan', '~> 1.12.0', '< 1.13.0'
 gem 'charlock_holmes', '~> 0.7.0'
 gem 'dalli', '~> 2.7.0'
 gem 'dynamic_form', '~> 1.1.0'
-gem 'exception_notification', '~> 4.0.0', '< 4.1.0'
+gem 'exception_notification', '~> 4.1.0', '< 4.1.2'
```

Comment:
I must have missed the fix in 4.1.1 (or - more likely - missed that 4.1.1 still had 1.9.3 support)